technology-adapter-api.md

```diff
@@ -26,6 +26,62 @@ Message:
 
 val - is error code, src - origin of the error.
 
+## Requesting status of a single device.
+
+An adapter has to support an API for requesting a device state.
+
+### Command
+
+Topic: `pt:j1/mt:cmd/rt:ad/rn:zigbee/ad:1`
+
+Message:
+
+```json
+{
+    "serv": "zigbee",
+    "type": "cmd.network.get_node",
+    "val_t": "int",
+    "val": 80,
+    "props": null,
+    "tags": null,
+    "ctime": "2018-11-22T23:14:40+0100",
+    "uid":"76533455876765"
```

Comment:
Please include complete message including "topic". Also uuid is string with hyphens, just copy random UUID and use it there in the example.

Reply:
what do you mean topic? it shouldn't be part of the payload too? Or do you mean the response topic?

btw, `uuidgen -r` is a nice command which should be already installed on most Debian/Ubuntu systems

Reply:
fixed

Reply:
Message has topic key as well. It's optional, but it should be there. It's for future compatibility of payloads if put on Kafka or other queues outiside of MQTT or IoT Core.